packages/remix-react/transition.ts

```diff
@@ -1215,10 +1282,18 @@ export function createTransitionManager(init: TransitionManagerInit) {
   }
 
   function abortNormalNavigation() {
-    pendingNavigationController?.abort();
+    if (pendingNavigationController) {
+      console.debug(`[transition] aborting pending navigation`);
+      pendingNavigationController.abort();
+      // TODO: Should this be cleared now so it doesn't stick around for
+      // subsequent navigations and make logs look weird?
+      // Or just remove this log?
+      // pendingNavigationController = undefined;
```

Comment:
@ryanflorence What do you think here?  I noticed that after our _first_ navigation this is non-null forever, so we seemed to always see this `aborting pending navigation` log - even for non-interruption-navigations.  I'm unsure if this is exactly the right place to reset though - or if it should be reset elsewhere?  It's not breaking anything really since calling abort subsequently shouldn't really do anything

Reply:
should probably be cleaned up after the navigation completes

Reply:
👍 Added in e475f236bf88070e204434d79452361199704230